src/pages/media/index_page.cr

```diff
@@ -0,0 +1,131 @@
+class Media::IndexPage < PageLayout
+  def hero_content
+    render_hero_content
+  end
+
+  def content
+    div class: "p-16" do
+      h1 "Assets", class: "text-2xl font-bold"
+
+      h2 "Logo & Icon", class: "mt-8 text-xl font-semibold"
+
+      div class: "mt-2 sm:flex items-center space-y-6 sm:space-y-0 sm:space-x-8" do
+        a href: asset("media/lucky_logo.png"), download: "lucky_logo.png", class: logo_wrapper_classes do
+          img src: asset("media/lucky_logo.png"), class: "p-4"
+          small "Logo", class: "text-white absolute bottom-2 left-2"
+        end
+
+        a href: asset("media/lucky_icon.png"), download: "lucky_icon.png", class: logo_wrapper_classes do
+          img src: asset("media/lucky_icon.png"), class: "p-4 h-24 w-24"
+          small "Icon", class: "text-white absolute bottom-2 left-2"
+        end
+      end
+
+      h2 "Colors", class: "mt-8 text-xl font-semibold"
+
+      div class: "mt-2 sm:flex sm:items-center space-y-6 sm:space-y-0 sm:space-x-8" do
+        render_color("Blue", "#031E37", "bg-lucky-blue")
+        render_color("Teal Blue", "#1B92B3", "bg-lucky-teal-blue")
+        render_color("Dark Green", "#37DE97", "bg-lucky-dark-green")
+        render_color("Light Green", "#9DFF66", "bg-lucky-light-green")
+      end
+
+      hr class: "my-16"
+
+      h1 "Guidelines", class: "text-2xl font-bold"
+
+      h2 "What can you do with the Lucky logos and brand?", class: "mt-8 mb-4 text-xl font-semibold"
+      ul class: "space-y-2" do
+        render_allowed_list_item("Print them on any merchandise you want, as long as you don’t sell it.")
+        render_allowed_list_item("Use them in presentations, papers and talks.")
+        render_allowed_list_item("Use them to advertise something you’ve created with Lucky.")
+        render_allowed_list_item("Use them in blog posts, articles or news about Lucky.")
+        render_allowed_list_item("Use them for educational purposes, as long as it’s available for free.")
+      end
+
+      h2 "Don’t do these things with the Lucky logos and brand:", class: "mt-8 mb-4 text-xl font-semibold"
+      ul class: "space-y-2" do
+        render_disallowed_list_item("Change their colors, dimensions or add your own text.")
+        render_disallowed_list_item("Sell Lucky artwork or merchandise without permission.")
+        render_disallowed_list_item("Integrate your own logo with Lucky logos.")
```

Comment:
Does this mean the LuckyCasts logo is in violation? 🤔  😛

Reply:
Haha yeah, that's what I was referencing in the description 😬 

Maybe I need to add "(except LuckyCasts pretty pls, see https://github.com/luckyframework/lucky/issues/23)" 😛

Reply:
Maybe “without permission” or add to the “contact us” part